owncloudData/src/test/java/com/owncloud/android/data/oauth/datasource/RemoteOAuthDataSourceTest.kt

```diff
@@ -97,10 +97,10 @@ class RemoteOAuthDataSourceTest {
             oidcService.performTokenRequest(ocClientMocked, any())
         } returns tokenResponseResult
 
-        val tokenResponsed = remoteOAuthDataSource.performTokenRequest(OC_TOKEN_REQUEST_ACCESS)
+        val tokenResponded = remoteOAuthDataSource.performTokenRequest(OC_TOKEN_REQUEST_ACCESS)
```

Comment:
This should probably be `tokenResponse` (corrections were automatically picked by Google Sheets)

Reply:
Yes, that's it. Could you rename it to `tokenResponse`?